### PR TITLE
Archipelago v2.3.2 - New Cask

### PR DESCRIPTION
**Pull request by andrewmcodes**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/archipelago.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'archipelago' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://archipelago-terminal.herokuapp.com/download/osx'
+  name 'Archipelago'
+  homepage 'https://archipelago-terminal.herokuapp.com/download/osx'
```
Comment (vitorgalvao) [inline Casks/archipelago.rb]: This is not a homepage, it’s a download link.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/archipelago.rb (hunk 1 above)
Comment (andrewmcodes) [reply Casks/archipelago.rb]: @vitorgalvao This application does not have a homepage and the linters really want one :unamused: Do you have any suggestions, other than submitting a PR to add a homepage of some sorts to the repo?

Comment (commitay) [reply Casks/archipelago.rb]: Where did you get the download URL from?

Comment (andrewmcodes) [reply Casks/archipelago.rb]: @commitay The [readme](https://github.com/npezza93/archipelago) of the repo.

Comment (commitay) [reply Casks/archipelago.rb]: https://github.com/npezza93/archipelago can be used as the homepage

File: Casks/archipelago.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'archipelago' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://archipelago-terminal.herokuapp.com/download/osx'
```
Comment (vitorgalvao) [inline Casks/archipelago.rb]: Make it a link from [the releases page](https://github.com/npezza93/archipelago/releases).

File: Casks/archipelago.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'archipelago' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/archipelago.rb]: Add a `version` and `sha256`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/archipelago.rb (hunk 3 above)
Comment (andrewmcodes) [reply Casks/archipelago.rb]: 👍

File: Casks/archipelago.rb (hunk 2 above)
Comment (andrewmcodes) [reply Casks/archipelago.rb]: 👍

File: Casks/archipelago.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cask 'archipelago' do
+  version '2.3.2'
+  sha256 '8bcc196c925c2f2de97cdaf816fd0b63f11b03fc0131d277449956b03bbdbd25'
+
+  url 'https://github.com/npezza93/archipelago/releases/tag/v2.3.2'
```
Comment (vitorgalvao) [inline Casks/archipelago.rb]: No. An actual dmg or zip. Something that actually downloads a package.

Comment (vitorgalvao) [reply Casks/archipelago.rb]: Why did you remove the `appcast` line? Don’t.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/archipelago.rb (hunk 4 above)
Comment (andrewmcodes) [reply Casks/archipelago.rb]: Sorry, I think everything should be good to go now.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**